pymets/mets_structure.py

```diff
@@ -78,7 +78,7 @@ def set_atts(self, attribute_dict):
                 raise MetsStructureException(
                     "Attribute %s is not legal in this element!" % (name,))
         # Remove empty attributes.
-        for key, value in self.atts.items():
+        for key, value in list(self.atts.items()):
```

Comment:
This is one of those changes suggested by 2to3 and similar tools that I just kinda ignore most of the time. `.items()` now returns an _iterable_ instead of an actual list, but when we iterate over the values in a for loop like this, there's no practical difference, so no real need to force the type to a list. Just adds a bit of complexity here with no gains.

Reply:
I think in this case we ant to use a list because there is a del from the source dict in the loop that could cause issues.

Reply:
My bad, we _do_ need that list wrapper. As you said, it's required since we're actually deleting things from the dict itself in the loop. Won't do to iterate over a dynamic view of the dict _while_ it's being modified.